qmri_mtsat/MTS.json

```diff
@@ -0,0 +1,7 @@
+{
+  "MagneticFieldStrength": "3",
+  "Manufacturer": "Siemens",
+  "ManufacturerModelName": "TrioTim",
+  "PulseSequenceType": "SPGR",
+  "RepetitionTime": "0.028"
```

Comment:
`RepetitionTime` may change along with FA. It is not the case for this one, but a possibility to keep in mind.